tests/functional/spec/components/radio/radio.spec.js

```diff
@@ -0,0 +1,161 @@
+const helpers = require('../../../helpers');
+
+describe('Component: Radio', function() {
+  describe('Given I start a Mandatory Radio survey', function() {
+
+    var RadioMandatoryPage = require('../../../pages/components/radio/mandatory/radio-mandatory.page');
+    var RadioSummaryPage = require('../../../pages/components/radio/mandatory/summary.page');
+
+    before('load the survey', function() {
```

Comment:
Can we drop all of the `'load the survey'` and just have `before(function() {`? I think it's obvious that we're loading the survey.